File: Java/src/main/java/com/semantria/mapping/output/DocCategory.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlElementWrapper;
import java.util.List;

public class DocCategory
{
    private String title = null;
    private String type = null;
    private Float sentiment_score = null;
    private Float strength_score = null;
    private String sentiment_polarity = null;

    private List<DocCategory> categories = null;

    @XmlElement(name="title")
    public String getTitle() { return title; }
    @XmlElement(name="type")
    public String getType() { return type; }
    @XmlElement(name="sentiment_score")
    public Float getSentimentScore() { return sentiment_score; }
    @XmlElement(name="strength_score")
    public Float getStrengthScore() { return strength_score; }
    @XmlElement(name = "sentiment_polarity")
    public String getSentimentPolarity() { return sentiment_polarity; }
    @XmlElementWrapper(name = "categories")
    @XmlElement(name = "category")
    public List<DocCategory> getCategories() { return categories; }

    public void setTitle(String title) { this.title = title; }
    public void setType(String type) { this.type = type; }
    public void setSentimentScore(Float sentiment_score) {this.sentiment_score = sentiment_score;}
    public void setStrengthScore(Float strength_score) { this.strength_score = strength_score; }
    public void setSentimentPolarity(String sentiment_polarity) { this.sentiment_polarity = sentiment_polarity; }
    public void setCategories(List<DocCategory> categories) { this.categories = categories; }
}

File: dotNet/Semantria.Com/Mapping/Output/DocCategory.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "category")]
    [XmlRootAttribute("category", Namespace = "")]
    public class DocCategory
    {
        [DataMember(Name = "title")]
        [XmlElementAttribute("title")]
        public string Title { get; set; }

        [DataMember(Name = "type")]
        [XmlElementAttribute("type")]
        public string Type { get; set; }

        [DataMember(Name = "strength_score")]
        [XmlElementAttribute("strength_score")]
        public float StrengthScore { get; set; }

        [DataMember(Name = "sentiment_score")]
        [XmlElementAttribute("sentiment_score")]
        public float SentimentScore { get; set; }

        [DataMember(Name = "sentiment_polarity")]
        [XmlElementAttribute("sentiment_polarity")]
        public string SentimentPolarity { get; set; }

        [DataMember(Name = "categories")]
        [XmlArrayAttribute("categories")]
        [XmlArrayItemAttribute("category", typeof(DocCategory))]
        public List<DocCategory> Categories { get; set; }
    }
}

